contracts/svm/programs/express_relay/src/lib.rs

```diff
@@ -248,9 +259,13 @@ pub struct SetSplitsArgs {
     pub split_relayer:        u64,
 }
 
+#[derive(AnchorSerialize, AnchorDeserialize, Eq, PartialEq, Clone, Copy, Debug)]
+pub struct SetSwapPlatformFeeArgs {
+    pub swap_platform_fee_bps: u64,
+}
+
 #[derive(Accounts)]
 pub struct SetSplits<'info> {
-    #[account(mut)]
```

Comment:
this doesn't need to be mutable imo

Reply:
i think you can remove the constraint from other places too, like `SetAdmin`, `WithdrawFees`, etc.